pkg/systemd/timer-dialog.jsx

```diff
@@ -154,31 +155,30 @@
             {dialogError && <ModalError dialogError={_("Timer creation failed")} dialogErrorDetail={dialogError} />}
             <Form isHorizontal onSubmit={onSubmit}>
                 <FormGroup label={_("Name")}
-                           fieldId="servicename"
-                           validated={submitted && validationFailed.name ? "error" : "default"}
-                           helperTextInvalid={!name.trim().length ? _("This field cannot be empty") : _("Only alphabets, numbers, : , _ , . , @ , - are allowed")}>
+                           fieldId="servicename">
                     <TextInput id='servicename'
                                value={name}
                                validated={submitted && validationFailed.name ? "error" : "default"}
                                onChange={setName} />
+                    <FormHelper fieldId="servicename"
+                                helperTextInvalid={submitted && validationFailed.name && (!name.trim().length ? _("This field cannot be empty") : _("Only alphabets, numbers, : , _ , . , @ , - are allowed"))} />
                 </FormGroup>
                 <FormGroup label={_("Description")}
-                           fieldId="description"
-                           validated={submitted && validationFailed.description ? "error" : "default"}
-                           helperTextInvalid={_("This field cannot be empty")}>
+                           fieldId="description">
                     <TextInput id='description'
                                value={description}
                                validated={submitted && validationFailed.description ? "error" : "default"}
                                onChange={setDescription} />
+                    <FormHelper fieldId="description" helperTextInvalid={submitted && validationFailed.description && _("This field cannot be empty")} />
                 </FormGroup>
                 <FormGroup label={_("Command")}
-                           fieldId="command"
-                           validated={submitted && validationFailed.command ? "error" : "default"}
-                           helperTextInvalid={commandNotFound ? _("Command not found") : _("This field cannot be empty")}>
+                           fieldId="command">
                     <TextInput id='command'
                                value={command}
                                validated={submitted && validationFailed.command ? "error" : "default"}
                                onChange={str => { setCommandNotFound(false); setCommand(str) }} />
+                    <FormHelper fieldId="command"
+                                helperTextInvalid={submitted && validationFailed.command && (commandNotFound ? _("Command not found") : _("This field cannot be empty"))} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18641-20230417-051207-efb10d39-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)